apis/v1alpha1/generator.yaml

```diff
@@ -1,10 +1,46 @@
 ignore:
   resource_names:
-  - ConfigurationSet
+#  - ConfigurationSet
   - ConfigurationSetEventDestination
   - CustomVerificationEmailTemplate
   - ReceiptFilter
   - ReceiptRule
   - ReceiptRuleSet
   - Template
+
+  field_paths:
+    - CreateConfigurationSetInput.ConfigurationSet
+
 model_name: email
+
+resources:
+  ConfigurationSet:
+    fields:
```

Comment:
It looks like`ConfigurationSetEventDestination` have a 1:1 mapping with `ConfigurationSet`, so it needs to be injected as a field here. We do similar things with S3::Bucket and S3::BucketPolicy (they are managed using one CRD), same with IAM:Role and IAM::RolePolicyAttachement

Reply:
And I believe we need to do the same with `ReceiptRuleSpec` and `ReceiptRule`